src/datagen/java/blusunrize/immersiveengineering/data/recipes/DecorationRecipes.java

```diff
@@ -192,11 +192,18 @@ private void stoneDecoration(RecipeOutput out)
 				.define('b', StoneDecoration.CLINKER_BRICK)
 				.unlockedBy("has_bricks", has(Blocks.BRICKS))
 				.save(out, toRL(toPath(StoneDecoration.CLINKER_BRICK_SILL)));
-		addSandwich(StoneDecoration.HEMPCRETE, 6,
-				makeIngredient(IETags.clay),
-				makeIngredient(IETags.fiberHemp),
-				makeIngredient(IETags.clay),
-				has(IETags.fiberHemp), out);
+		ShapedRecipeBuilder.shaped(RecipeCategory.MISC, StoneDecoration.HEMPCRETE, 8)
+			.pattern("scs")
+			.pattern("tbt")
+			.pattern("scs")
+			.define('s', Tags.Items.SAND)
+			.define('c', IETags.clay)
+			.define('t', Ingredients.HEMP_FABRIC)
+			.define('b', new IngredientFluidStack(FluidTags.WATER, FluidType.BUCKET_VOLUME))
+			.unlockedBy("has_clay", has(IETags.clay))
+			.save(new WrappingRecipeOutput<ShapedRecipe>(
+				out, r -> new TurnAndCopyRecipe(r, List.of()).allowQuarterTurn()
+			), toRL("hempcrete"));
```

Comment:
Why is this in here, this doesn't seem to have anything to do with the manual?

Reply:
Ooop, will remove!